apps/omg_api/lib/state/core.ex

```diff
@@ -233,7 +234,7 @@ defmodule OMG.API.State.Core do
     fees_covered =
       for {input_currency, input_amount} <- Map.to_list(input_amounts) do
         output_amount = Map.get(output_amounts, input_currency, 0)
-        fee = Map.get(fees, input_currency, 0)
+        fee = Map.get(fees, input_currency, :infinity)
```

Comment:
hey... that was just a figure of speech, this is dangerous. Just `case` against a `nil` here to return an explicit `false` on `nil`